pkgs/by-name/pi/pixiefairy/package.nix

```
@@ -0,0 +1,42 @@
+{
+  python3Packages,
+  nix-gitignore,
+  fetchFromGitHub,
+  lib,
+}:
+python3Packages.buildPythonApplication {
+  pname = "pixiefairy";
+  version = "0.2.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "xhalo32";
+    repo = "pixiefairy";
+    rev = "e5cd76af9d92dfb6db9adedd4dc58f9772785fe7";
```

```suggestion
   tag = "v${version}";
```
since you have tags in your fork

---

Updated